bin/udmi-pubber

```diff
@@ -1,18 +1,12 @@
 #!/bin/bash -e
 
```

Comment:
Is there some reason these aren't named same_as_the_others instead of using - ?

Reply:
I was basing it off of the git extension format, e.g. the file "git-utility" would map to the command-line "git utility " -- as a way to provide transparent extensions. So, "udmi-validate" maps to "udmi validate".  The snake_case ones are generally things then that should be run directly (bin/test_sequencer).  So, that was the reason (consistency with git extension naming)...